user-guide/Advanced_Functionality/Security/About_DMS_Security/Cloud_connectivity_and_security.md

```diff
@@ -0,0 +1,44 @@
+
+It is important to note that users can disconnect their system from the DataMiner Cloud Platform at any given time. For more information, please refer to [Connecting your DataMiner System to the cloud](xref:Connecting_your_DataMiner_System_to_the_cloud).
+
+For further technical details, please refer to [Cloud Connection Security](xref:CloudConnectionSecurity).
+
+## Live Sharing
+
+Users can share a dashboard by clicking the *Start sharing* button in the dashboard menu bar. They can then specify a list of email addresses to share the dashboard with. Those email addresses will receive a link to the shared dashboard.
+
+When the recipients open the link, they will need to authenticate by logging in with the account on the DataMiner Cloud Platform linked to that email address, or by creating a dedicated DataMiner Cloud Platform account if they don't have any of the formerly mentioned accounts. After successfully authenticating, the recipient will be directed to the dashboard through the DataMiner Cloud Platform tunnel.
+
+No data from the private DataMiner System hosting the dashboard is stored on the DataMiner Cloud Platform. The share can be deleted in the same way as it was created. All emailed links will then automatically become unreachable. For more information, please refer to [Sharing a dashboard](xref:Sharing_a_dashboard).
+
+Behind the scenes, DataMiner relies on several defense mechanisms to prevent unauthorized access to the shared dashboards.
+
+### Cloud users
+
+When a dashboard is shared, DataMiner will generate a special Cloud user (see [Types of users](xref:Types_of_users)), which is used to view the shared dashboards. These Cloud users have randomly generated passwords (17 characters) and can only be used through the DataMiner Cloud Gateway. The passwords for these users are encrypted using the Blowfish algorithm (308-bit key, ECB mode) before being stored. These users are only granted permissions on the items in the shared dashboard. For example, if your dashboard contains 2 elements, the resulting Cloud user for that shared dashboard will only have read access to those 2 elements. Even if the user sharing the dashboard has access to all elements in the system.
+
+### Web Application Firewall (WAF)
+
+When a dashboard is shared, DataMiner will build a whitelist based on the API calls (and their parameters) of the dashboard. For example, consider your dashboard is displaying all active alarms for element 'SOC' with Element ID 100 and DataMiner ID 2000. When the dashboard is shared, DataMiner will infer it calls the GetActiveAlarmsForElement API with parameters Element ID 100 and DataMiner ID 200. If that shared dashboard connection would suddenly request the active alarms for a different element, or even a different API call, DataMiner will block the request.
```

Comment:
Avoid using "would" in an if-clause: "If that shared dashboard connection were to suddenly request [...], DataMiner would block the request"